data/tutorials/lg_08_error_handling.md

```diff
@@ -125,67 +191,549 @@ let () =
     )
 ```
 
-Each printer should take care of the exceptions it knows about, returning
-`Some <printed exception>`, and return `None` otherwise (let the other printers
-do the job!).
+Each printer should take care of the exceptions it knows about, returning `Some
+<printed exception>`, and return `None` otherwise (let the other printers do the
+job!).
+
+## Runtime Crashes
+
+Although OCaml is a very safe language, it is possible to trigger unrecoverable
+errors at runtime.
+
+### Exceptions Not Raised
+
+Under panic circumstances, the native code compiler gives its best effort for
+raising meaningful exceptions. However, some error conditions may remain
+undetected, which will result in a segmentation fault. This is the specially the
+case for stack overflows, which aren't always detected.
+
+> But catching stack overflows is tricky, both in Unix-like systems and under
+> Windows, so the current implementation in OCaml is a best effort that is
+> occasionally buggy.
+
+[Xavier Leroy, October
+2021](https://discuss.ocaml.org/t/stack-overflow-reported-as-segfault/8646/8)
+
+### Bypassing Type-Safety
+
+OCaml provides means to bypass its type safety. Don't use it. Here is how
+to shoot oneself in the foot:
+
+```shell
+> echo "(Obj.magic () : int array).(0)" > foo.ml
+> ocamlopt foo.ml
+> ./a.out
+Segmentation fault (core dumped)
```

Comment:
I get a Syntax error here if I put ``;;`` at the end, and it just doesn't give an output without it. 
Also, the command prompt is denoted as a `>` here whereas earlier in the document it's `#`
Should they be the same?

Reply:
Will that be clear to the reader about the Unix shell?

Reply:
Yes, it trust they should be.